modules/index/companies-using-kedro/companies-using-kedro.module.scss

```
@@ -16,57 +16,57 @@
 }
 
 .inner {
-  background: $color-bg-grey;
-  flex-direction: column;
-  font-weight: 700;
-  line-height: 6rem;
   display: flex;
   margin: 0 auto;
-  max-width: $max-content;
-
-  @include breakpoint(small) {
-    font-size: 1.5rem;
-    line-height: 3rem;
-  }
 }
 
-.title {
-  @include h1;
-
-  margin-bottom: 6rem;
-  text-align: center;
-
-  @include breakpoint(small) {
-    margin-bottom: 2.5rem;
+.titleContainer {
+  white-space: nowrap;
 
-    padding: 0;
+  .title {
+    @include display-md;
+    line-height: 7rem;
+    font-weight: 500;
+    padding-right: 2em;
+    @include breakpoint(small) {
```

Try and have a blank line before `@` rules.

```suggestion
    padding-right: 2em;
    
    @include breakpoint(small) {
```